### ISSUE TITLE
Does this work with self-hosted LanguageTool instances?

### ISSUE THREAD

**Issue by readtedium** [tito21/joplin-langugetool#4]
The docs don’t seem to mention it, but LanguageTool [works well with Docker](https://github.com/Erikvl87/docker-languagetool) and such.

**Comment by kieric82** [comment]
> The docs don’t seem to mention it, but LanguageTool [works well with Docker](https://github.com/Erikvl87/docker-languagetool) and such.

Confirmed on my side, just change the API web address and should be it.

**Comment by tito21** [comment]
Yeah, there is not documented, but it should work by changing the URL in the settings.

If you can confirm it works, I can close the issue.

**Comment by readtedium** [comment]
Honestly, it isn’t working at all, to be honest. I tried in dev mode and saw this error with a right click:

![Screenshot from 2024-03-14 16-38-24](https://github.com/tito21/joplin-langugetool/assets/470216/4336375a-3139-4d82-80c5-9bb513dbe5e7)

The right-click menu does not function at all, and there are no visual signs of the plugin. Turning off the plugin revives the right-click menu.

(Using a Flatpak version of Joplin in Fedora if it helps. Have not tested an RPM version or anything similar.)

**Comment by readtedium** [comment]
Same problem with the official AppImage.

**Comment by k-electric-ger** [comment]
I'd love to have this feature added!

**Comment by LeslieFH** [comment]
I was very happy about the plugin, I installed it, set the URL to http://localhost:8081/v2/check and it doesn't work, neither does http://192.168.1.1:8081/v2/check 

My locally hosted LanguageTool works perfectly with the Firefox plugin and other software that supports languagetool, it's a pity it doesn't work with Joplin.
